modules/ding_library/ding_library.features.field_instance.inc

```diff
@@ -753,7 +753,7 @@ function ding_library_field_default_field_instances() {
         'label' => 'above',
         'settings' => array(),
         'type' => 'hidden',
-        'weight' => 13,
+        'weight' => 8,
```

Comment:
Note to self: These changes of weight have not practical consequence. The order will remain the same.